desktop/core/src/desktop/lib/raz/raz_client.py

```diff
@@ -71,25 +71,36 @@ def renew_delegation_token(self, user):
 
 class RazClient(object):
 
-  def __init__(self, raz_url, raz_token, username, service='s3', service_name='cm_s3', cluster_name='myCluster'):
+  def __init__(self, raz_url, raz_token, username, service='s3', service_name='cm_s3', cluster_name='myCluster', operation={}):
     self.raz_url = raz_url.strip('/')
     self.raz_token = raz_token
     self.username = username
     self.service = service
+
     if self.service == 'adls':
       self.service_params = {
         'endpoint_prefix': 'adls',
         'service_name': 'adls',
         'serviceType': 'adls'
       }
+      self.client_params = {
+        "clientIpAddress": "12.12.12.12",
```

Comment:
We probably can keep it simpler by not doing yet another dict and we don't care about the clientIpAddress I bet